tests/components/deconz/test_device_trigger.py

```diff
@@ -88,51 +78,51 @@ def _same_lists(a, b):
     """Test triggers work."""
     gateway = await setup_deconz(hass, options={})
     device_id = gateway.events[0].device_id
-    triggers = await async_get_device_automations(hass, "async_get_triggers", device_id)
+    triggers = await async_get_device_automations(hass, "trigger", device_id)
 
     expected_triggers = [
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_SHORT_PRESS,
-            "subtype": deconz.device_automation.CONF_TURN_ON,
+            "type": device_trigger.CONF_SHORT_PRESS,
+            "subtype": device_trigger.CONF_TURN_ON,
         },
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_LONG_PRESS,
-            "subtype": deconz.device_automation.CONF_TURN_ON,
+            "type": device_trigger.CONF_LONG_PRESS,
+            "subtype": device_trigger.CONF_TURN_ON,
         },
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_LONG_RELEASE,
-            "subtype": deconz.device_automation.CONF_TURN_ON,
+            "type": device_trigger.CONF_LONG_RELEASE,
+            "subtype": device_trigger.CONF_TURN_ON,
         },
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_SHORT_PRESS,
-            "subtype": deconz.device_automation.CONF_TURN_OFF,
+            "type": device_trigger.CONF_SHORT_PRESS,
+            "subtype": device_trigger.CONF_TURN_OFF,
         },
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_LONG_PRESS,
-            "subtype": deconz.device_automation.CONF_TURN_OFF,
+            "type": device_trigger.CONF_LONG_PRESS,
+            "subtype": device_trigger.CONF_TURN_OFF,
         },
         {
             "device_id": device_id,
             "domain": "deconz",
             "platform": "device",
-            "type": deconz.device_automation.CONF_LONG_RELEASE,
-            "subtype": deconz.device_automation.CONF_TURN_OFF,
+            "type": device_trigger.CONF_LONG_RELEASE,
+            "subtype": device_trigger.CONF_TURN_OFF,
         },
     ]
 
-    assert _same_lists(triggers, expected_triggers)
+    assert triggers == expected_triggers
```

Comment:
This assumes the lists have the same order, isn't that too restrictive?

Reply:
Tests pass? 👍

Reply:
Yeah, OK.